heat/regression/lasso.py

```diff
@@ -48,12 +56,17 @@ class Lasso(ht.RegressionMixin, ht.BaseEstimator):
     """
 
     def __init__(
-        self, lam: Optional[float] = 0.1, max_iter: Optional[int] = 100, tol: Optional[float] = 1e-6
+        self,
+        lam: Optional[float] = 0.1,
+        max_iter: Optional[int] = 100,
+        tol: Optional[float] = 1e-6,
+        step_size: Optional[float] = None,
```

Comment:
Are lam, max_iter, and tol allowed to be None? The docstring doesn't mention it.
Since python 3.10, you can also use `float | None` instead of `Optional[float]` if it confuses you.